README.md

```diff
@@ -1,11 +1,11 @@
-[![Circle CI](https://circleci.com/gh/Financial-Times/pac-annotations-mapper.svg?style=shield)](https://circleci.com/gh/Financial-Times/pac-annotations-mapper) [![Coverage Status](https://coveralls.io/repos/github/Financial-Times/pac-annotations-mapper/badge.svg)](https://coveralls.io/github/Financial-Times/pac-annotations-mapper)
-
 # PAC Annotations Mapper
 
+[![Circle CI](https://circleci.com/gh/Financial-Times/pac-annotations-mapper.svg?style=shield)](https://circleci.com/gh/Financial-Times/pac-annotations-mapper) [![Coverage Status](https://coveralls.io/repos/github/Financial-Times/pac-annotations-mapper/badge.svg)](https://coveralls.io/github/Financial-Times/pac-annotations-mapper)
+
 Mapper for transforming PAC annotations to UPP annotations
 
 * Reads PAC metadata for an article from the Kafka topic _NativeCmsMetadataPublicationEvents_
-	* Metadata for sources different than "http://cmdb.ft.com/systems/pac" is skipped by this mapper
+* Metadata for sources different than "<http://cmdb.ft.com/systems/pac"> is skipped by this mapper
```

Comment:
I'm wondering why we need <> here and why we need quotation marks at all.

Reply:
<> came from my markdown linter https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md034---bare-url-used

Apparently it did not detect the correct string as url.

Reply:
OK, but in this case the > sign should be placed before the second quotes.

Reply:
OK, but in this case the closing > should be placed before the second quotes IMO.

Reply:
It's fixed already